src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+public class Parser {
+
+    /**
+     * Returns specific Duke.Duke.Command from given user input.
+     * @param fullCommand User input string.
+     * @return Duke.Duke.Command to be executed.
+     */
+    public static Command parse(String fullCommand) {
+        String[] splitInput = fullCommand.split(" ");
+        String command = splitInput[0];
```

Comment:
Code looks neat here and very organised, good as well adhering to the coding style for this particular case